src/lib/common/errorMessages.h

```diff
@@ -76,6 +76,14 @@
 #define ERROR_DESC_BAD_REQUEST_ENTID_IN_PAYLOAD               "entity id specified in payload"
 #define ERROR_DESC_BAD_REQUEST_ENTTYPE_IN_PAYLOAD             "entity type specified in payload"
 #define ERROR_DESC_BAD_REQUEST_NOT_A_JSON_OBJECT              "not a JSON object"
+#define ERROR_DESC_BAD_REQUEST_SUBSCRIPTIONID_NOT_STRING      "/subscriptionId/ must be a JSON string"
+#define ERROR_DESC_BAD_REQUEST_DATA_NOT_ARRAY                 "notification field /data/ must be a JSON array"
+#define ERROR_DESC_BAD_REQUEST_DATA_ITEM_NOT_OBJECT           "notification data vector item must be a JSON object"
+#define ERROR_DESC_BAD_REQUEST_NO_SUBSCRIPTION_ID             "invalid JSON payload, mandatory field /subscriptionId/ not found"
+#define ERROR_DESC_BAD_REQUEST_FORMAT_KEYVALUES               "attribute render format /keyValues/ not yet supported for notifications"
```

Comment:
not yet supported -> not supported (also in the others)

"yet" gives the idea that it is not supported now but it will be supported in the future, which I think is very improbable (at least for uniqueValues and custom, which miss information)

Reply:
ok, I thought we would actually implement that more or less soon ...